cmake/CeleritasOptionUtils.cmake

```
@@ -168,16 +173,19 @@ macro(celeritas_find_or_builtin_package package)
       # Celeritas is a subproject)
       find_package(${package} ${ARGN})
     endif()
+    set(_found "${${package}_FOUND}")
```

Since this is macro, it might be helpful to obfuscate the name more.
```suggestion
    set(_celeritas_builtin_found "${${package}_FOUND}")
```
(and of course the other related changes)

---

These local variables are "unset" immediately after. You'll see the same idiom in official `FindX.cmake` packages too.

---

hummm ... If somebody make the 'mistake' to write:
```
set(_found SomethingClever)
celeritas_find_or_builtin_package(....)
message(STATUS "Is ${_found} still something clever")?
```

---

Then it doesn't show anything, because `_found` is unset.

---

exactly my point :).  In my opinion, it should not, the caller of `celeritas_find_or_builtin_package` should be expected to know the internal variable used and wiped by `celeritas_find_or_builtin_package` :)